### PR TITLE
adding capability for string based entry file

### LINKED ISSUE #17: Crashes when entry is a string
Hi! First of all, this plugin is awesome! )

I have played a little bit and found that it doesn't handle case when the entry is a string but not an object.

Example:

```js
{
 // webpack configuration
 entry: './index.js'
}
```

<details>
<summary>Error message</summary>
    ERROR in Entry module not found: Error: Can't resolve 'd' in '/test/src-example'
    
    ERROR in Entry module not found: Error: Can't resolve 'e' in '/test/src-example'
    
    ERROR in Entry module not found: Error: Can't resolve 'i' in '/test/src-example'
    
    ERROR in Entry module not found: Error: Can't resolve 'n' in '/test/src-example'
    
    ERROR in Entry module not found: Error: Can't resolve 'x' in '/test/src-example'
</details>

### PR DESCRIPTION
**Pull request by prateekbh**
fixes #17

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -66,18 +66,26 @@ class BabelEsmPlugin {
...
+            new MultiEntryPlugin(compiler.context, entryFiles, entry).apply(
+              childCompiler,
+            );
+          } else {
+            new SingleEntryPlugin(compiler.context, entryFiles, entry).apply(
+              childCompiler,
+            );
+          }
+        });
+      }
```
Comment (ForsakenHarmony) [inline src/index.js]: how about merging the branches here?

i.e.

`if (typeof compiler.options.entry === 'string') compiler.options.entry = { index: compiler.options.entry };`

possibly extracting the var first

Comment (prateekbh) [reply src/index.js]: done

Comment (ForsakenHarmony) [reply src/index.js]: as long as overriding it doesn't affect something

Comment (prateekbh) [reply src/index.js]: i guess the tests cover that, right?

Comment (prateekbh) [reply src/index.js]: @AlexMost WDYT?

Comment (AlexMost) [reply src/index.js]: looks good, thanks!

**Pull request merged**